### PR TITLE
Fix: resolve offchip logic that was broken

### PR DESCRIPTION
**Pull request by madcampos**
- Remove unreliable and unimplemented `isOffchip` property.
- Move logic for offchip verification to other places, specifically a check between the current chip and the operation chip.

### CODE REVIEW

File: src/data/GraphOnChip.ts [hunk 1]
```diff
@@ -659,7 +659,7 @@ export default class GraphOnChip {
                     queue.assignInputs([newChip.createOperand(opName, GraphVertexType.OPERATION)]);
                 } else if (newChip.operationsByName.has(operand.name)) {
                     const op = newChip.operationsByName.get(operand.name);
-                    if (op?.isOffchip) {
+                    if (op?.cores.length === 0) {
```
Comment (aidemsined) [inline src/data/GraphOnChip.ts:662]: you cant call a property thats actively set and used "uneeded and unimplemented"

you shoudl not transfer operation logic to graph on chip

**Review by aidemsined — CHANGES_REQUESTED** [review]
please retain offchip calculation logic in the operand.

File: src/data/GraphOnChip.ts (hunk 1 above)
Comment (madcampos) [reply src/data/GraphOnChip.ts:662]: The abstract class defined it and there was one implementation, on other places it was either not implemented or returned `false`, totally defeating the point of having this property in the first place.

And this property was only used in one place, that is why I moved the logic there.

**Review by aidemsined — CHANGES_REQUESTED** [review]
this is clearly not working correctly. i see no "offchip" indicators
image

cores view is also not working as expected
image

**Pull request merged**